controllers/projectController.js

```diff
@@ -61,16 +66,14 @@ const controller = {
       // Get comments TODO consolidate user details into comments
       comments = await CommentModel.find({ project_id: project.user_id }).lean();
       // Get jobs
-      // eslint-disable-next-line no-underscore-dangle
       jobs = await ContributorModel.find(
```

Comment:
should we also name the variables as 'contributors' instead of 'jobs'?
just to keep it consistent

Reply:
I thought jobs were the actual jobs and contributors were the people?